components/compliance-service/reporting/relaxting/suggestions.go

```diff
@@ -539,7 +539,7 @@ func (backend ES2Backend) getControlSuggestions(ctx context.Context, client *ela
 	var innerQuery elastic.Query
 	if len(text) >= 2 {
 		innerBoolQuery := elastic.NewBoolQuery()
-		innerBoolQuery.Must(elastic.NewMatchQuery(fmt.Sprintf("%s.engram", target), text).Operator("or"))
+		innerBoolQuery.Must(elastic.NewMatchQuery(fmt.Sprintf("%s.engram", target), text).Operator("and"))
```

Comment:
This change does make the tests pass, but I am not sure if it is correct. The problem is with ES7 the "or" is not filtering enough values.